Tensile/Common.py

```diff
@@ -1292,6 +1292,7 @@ def getArchitectureName(gfxName):
     "MinVgprNumber":                list(range(0,256)),
 
     "MaxVgprNumber":                list(range(0,257)),
+    "MinKForGSU":                   range(1,1024)
```

Comment:
Shouldn't this be a list?

Reply:
Please add some comment here to show the usage.
(at least similar comment as the previous global parameter)